projects/cdk/directives/control/control.directive.ts

```diff
@@ -10,7 +10,7 @@ import {NgControl} from '@angular/forms';
 export class TuiControlDirective {
     private readonly ngControl = inject(NgControl);
 
-    protected get control(): AbstractControl {
+    public get control(): AbstractControl {
```

Comment:
why public ?

Reply:
That is, most likely it was assumed that this directive could be injected and access the control